starter-code/iteration-2/main.ts

```diff
@@ -1,15 +1,40 @@
 // Add the reference to the "TodoInterface"
 
 // 1. Create a class Todo that implements the Interface created before.
+import { todoInterface } from './todoInterface';
 
-// Execution
-let myTodos = new Todo();
-console.log("Number of items:", myTodos.addTask('This is our first task'));
-console.log("Number of items:", myTodos.addTask('Eat pizza 🍕 yummy!!!'));
-console.log("Number of items:", myTodos.addTask('Finish this iteration 1!! 🤓'));
-console.log("Number of items:", myTodos.addTask('Finish this iteration 2!! 🤓'));
-console.log("Number of items:", myTodos.addTask('Finish this iteration 3!! 🤓'));
-myTodos.listAllTasks();
-console.log("Number of items:", myTodos.deleteTask('Finish this iteration 1!! 🤓'));
-console.log("Number of items:", myTodos.deleteTask('Finish this iteration 2!! 🤓'));
-myTodos.listAllTasks();
+class Todo implements todoInterface {
+
+    constructor(public taskList: Array<String> = []) { }
+    addTask(task): Number {
```

Comment:
Caution! Use 'number' instead of 'Number'.
Furthermore, you missed the type of task (task: string).